File: app/src/main/java/com/example/mvvm_artichecture_sample/base/network/statuscode/ClientCode.java
package com.example.mvvm_artichecture_sample.base.network.statuscode;

public class ClientCode {

    public static final int NO_INTERNET_TIMEOUT_CODE = 700;
    public static final int SERVER_TIMEOUT_CODE = 701;

    public static final int EXCEPTION_CODE = -1;


    public static boolean isExceptionError(int code){
        return (code ==NO_INTERNET_TIMEOUT_CODE||code==SERVER_TIMEOUT_CODE);
    }
}

File: app/src/main/java/com/example/mvvm_artichecture_sample/base/network/statuscode/ClientCode.kt
package com.example.mvvm_artichecture_sample.base.network.statuscode

object ClientCode {

    const val NO_INTERNET_TIMEOUT_CODE = 700
    const val SERVER_TIMEOUT_CODE = 701

    const val EXCEPTION_CODE = -1


    fun isExceptionError(code: Int): Boolean {
        return code == NO_INTERNET_TIMEOUT_CODE || code == SERVER_TIMEOUT_CODE
    }
}
